### PR TITLE
West doc fixes

### PR DESCRIPTION
**Pull request by mbolivar**
Updates from a periodic review of existing documentation.

### CODE REVIEW

File: doc/guides/west/repo-tool.rst [hunk 1]
```diff
@@ -205,11 +205,11 @@ important to understand.
      revision using ``--mr``. For example, invoking west with: ``west init -m
      https://github.com/zephyrproject-rtos/zephyr --mr v1.15.0`` would clone
      the upstream official zephyr repository at the tagged release v1.15.0
-     (``-m`` defaults to https://github.com/zephyrproject-rtos/zephyr, and
-     ``--mr`` defaults to ``master``).
+     (``-m`` defaults to https://github.com/zephyrproject-rtos/zephyr``, and
```
Comment (carlescufi) [inline doc/guides/west/repo-tool.rst]: Missing double backtick before `https://`

**Review by carlescufi — CHANGES_REQUESTED** [review]

**Review by dbkinder — COMMENTED** [review]
Fix @carlescufi's note about missing double backticks and it looks OK

File: doc/guides/west/install.rst [hunk 2]
```diff
@@ -15,32 +15,13 @@ On Windows and macOS::
...
-* macOS:
-
-  * Executable: :file:`/usr/local/bin/west`
-  * Package: :file:`/usr/local/lib/python3.<x>/site-packages/west/`
-
-* Windows:
-
-  * Executable: :file:`C:\\Python3<x>\\Scripts\\west.exe`
-  * Package: :file:`C:\\Python3<x>\\Lib\\site-packages\\west\\`
-
```
Comment (marc-hb) [inline doc/guides/west/install.rst]: Did you move these paths somewhere else in a different commit? Either way the commit message should say. The other changes are minor enough not to need a mention in the commit message, it's just this one.

File: doc/guides/west/repo-tool.rst (hunk 1 above)
Comment (mbolivar) [reply doc/guides/west/repo-tool.rst]: I'm going to treat this as "extra backticks after zephyr", i.e. remove the trailing backticks so the link renders.

File: doc/guides/west/install.rst (hunk 2 above)
Comment (mbolivar) [reply doc/guides/west/install.rst]: Sure, I'll update the commit message describing what happened here. This was a change that was requested in review of the original commit adding this file that didn't happen.

**Comment by mbolivar** [comment]
@carlescufi @marc-hb @dbkinder comments addressed; thanks for review!

**Review by carlescufi — APPROVED** [review]
